sdk/storage/Azure.Storage.Files.DataLake/src/Models/GetFileSystemsAsyncCollection.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Azure.Storage.Files.DataLake.Models
 {
-    internal class GetFileSystemsAsyncCollection : StorageCollectionEnumerator<FileSystemItem>
+    internal class GetFileSystemsAsyncCollection
```

Comment:
Add a comment that this wraps the BlobServiceClient.GetContainersAsync return values and maps them into DataLake types